.idea/.gitignore

```diff
@@ -0,0 +1,8 @@
+# Default ignored files
```

Comment:
The top level .gitignore is ok to check in. But maybe better not to include another one in an IDE-specific subdirectory like ".idea"

Reply:
Agreed